app/src/main/java/com/markkolenbrander/capstonenewsapp/views/ArticleView.kt

```diff
@@ -0,0 +1,33 @@
+package com.markkolenbrander.capstonenewsapp.views
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.markkolenbrander.capstonenewsapp.databinding.ArticleViewBinding
+import com.markkolenbrander.capstonenewsapp.models.Article
+
+class ArticleView @JvmOverloads constructor  (
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0,
+) : ConstraintLayout(context, attrs, defStyleAttr, defStyleRes) {
+
+    private val binding = ArticleViewBinding.inflate(LayoutInflater.from(context), this )
+
+    fun setArticleData(article: Article?){
```

Comment:
Just a nitpick, but you could probably enforce the `Article` here instead of using `Article?`.  That way you won't have to "unwrap" the `article` variable on each of the assignments below.

Reply:
True, I will change it